src/modules/db_operations/seed_trait_files.py

```diff
@@ -13,6 +13,31 @@
 ANDERSEN_LAB_TRAIT_SHEET = get_secret(f'ANDERSEN_LAB_TRAIT_SHEET')
 
 
+# Set of column headers that must exist in the sheet
+# NOTE: Not every entry needs to define all of these fields -- these are just the columns that should be available in the sheet.
+REQUIRED_SHEET_HEADERS = frozenset({
+  'Trait_Name_CaeNDR',
+  'Trait_Name_User',
+  'Species',
+  'Trait_Name_Display1',
+  'Trait_Name_Display2',
+  'Trait_Name_Display3',
+  'Short_Description',
+  'Long_Description',
+  'Units',
+  'Publication',
+  'Protocol',
+  'Institution',
+  'Captured_By_UserID',
+  'Capture_Date',
+
+  # TODO: How should we handle category tags?
```

Comment:
we don't expect these to change too often, but when they do, it would be nice to not have to make code changes to support it.